apps/i18n/applab/en_us.json

```diff
@@ -18,8 +18,133 @@
   "designToolboxHeader": "Design Toolbox",
   "designWorkspaceDescription": "Click on an element or choose it from the dropdown below to edit its properties. Create code in Events.",
   "designWorkspaceHeader": "Workspace",
+  "designWorkspace_copyToScreenButton": "Copy to screen",
+  "designWorkspace_deleteButton": "Delete",
+  "designWorkspace_deleteConfirm": "Delete?",
+  "designWorkspace_duplicateButton": "Duplicate",
+  "designWorkspace_eventInsertButton": "Insert and show code",
+  "designWorkspace_makeDefaultButton": "Make Default",
+  "designWorkspace_restoreThemeButton": "Apply Theme",
+  "designWorkspace_eventsTab": "EVENTS",
+  "designWorkspace_propertiesTab": "PROPERTIES",
+  "designElement_button": "Button",
+  "designElement_button_clickEvent": "Click",
+  "designElement_button_clickEventDesc": "Triggered when the button is clicked with a mouse or tapped on a screen.",
+  "designElement_canvas": "Canvas",
+  "designElement_canvas_clickEvent": "Click",
+  "designElement_canvas_clickEventDesc": "Triggered when the canvas is clicked with a mouse or tapped on a screen.",
+  "designElement_chart": "Chart",
+  "designElement_chart_drawEvent": "drawChart",
+  "designElement_chart_drawEventDesc": "Draws the chart using data you provide.",
+  "designElement_chart_drawFromRecordsEvent": "drawChartFromRecords",
+  "designElement_chart_drawFromRecordsEventDesc": "Draws the chart using App Lab's table data storage.",
+  "designElement_checkbox": "Checkbox",
+  "designElement_checkbox_changeEvent": "Change",
```

Comment:
It's a great idea to group them under the same key, but I'm not sure about moving them to the `common` translations since this is a specific event name (even if it uses a common word)